### ISSUE TITLE
Add a store method to spawn effect

### ISSUE THREAD

**Issue by EquipableMC** [SkriptLang/Skript#6792]
### Suggestion

I feel like you should be able to do (optionally) `spawn x and store it in {_var}` without having to do `set {_x} to last spawned entity` after the spawn effect. 

### Why?

What this does is, is spawn whatever x is and it will be saved in that variable so u can use it in the rest of your code. This is similar to how DiSky does their store stuff in their own addon, which I find pretty useful

### Agreement

- [X] I have read the guidelines above and affirm I am following them with this suggestion.

**Comment by Phill310** [comment]
Why not use spawn sections instead

**Comment by ShaneBeee** [comment]
1) I agree with Phill, the spawn section is what should be used here
2) While I understand Disky (and other addons do this) in my opinion it doesn't look pretty and it doesn't really flow with Skript

tl;dr:
- spawn section
- last spawned entity expression

**Comment by Moderocky** [comment]
We don't really want to introduce this bad design of syntax, the spawn section should be able to help you with this since you can use its local event entity.

**Comment by EquipableMC** [comment]
Fair enough, I understand
